spec/acceptance/class_logstash_spec.rb

```diff
@@ -1,40 +1,41 @@
+# frozen_string_literal: true
+
 require 'spec_helper_acceptance'
 
 shared_examples 'a logstash installer' do
-  it "should install logstash version #{LS_VERSION}" do
+  it "installs logstash version #{LS_VERSION}" do
     expect(shell('/usr/share/logstash/bin/logstash --version').stdout).to eq("logstash #{LS_VERSION}\n")
   end
 
   case fact('osfamily')
   when 'RedHat', 'Suse'
     describe package('logstash') do
-      it { should be_installed }
+      it { is_expected.to be_installed }
     end
   when 'Debian'
     # Serverspec has been falsely reporting the package as not installed on
     # Debian 7, so we'll implement our own version of "should be_installed".
```

Comment:
Is this still an issue or has it been resolved? Can we simply move to common `package()`?